### DISCUSSION TITLE
Move existing content item to another content item with list

### DISCUSSION THREAD

**Discussion by sergeor** [OrchardCMS/OrchardCore#10785] [Q&A] [answered]
I have 5 content items named "Article" and then I create an "Articles" content items with list of "Article". How can I move each of the 5 article in the Articles list without recreating them?

**Comment by Skrypt** [comment, marked as answer]
You can't do this from the admin UI right now. Maybe by editing the database directly.

**Reply by ShaneCourtrille** [reply]
We've seen problems with trying to modify the database directly while the site is running so if you go this route I'd look at taking it down.
